align/pnr/toplevel.py

```diff
@@ -243,6 +327,27 @@ def PnRdatabase( path, topcell, vname, lefname, mapname, drname):
 
     return DB
 
+def PnRdatabaseJson( path, topcell, vname, lefname, mapname, drname):
```

Comment:
I assume this is meant to be a drop-in replacement for toplevel.PnRdatabase eventually?